helm/prometheus-meta-operator/templates/prometheus-rules/management-cluster.rules.yml

```diff
@@ -46,3 +77,39 @@ spec:
         severity: page
         team: biscuit
         topic: managementcluster
+    - alert: ManagementClusterPodLimitAlmostReached
+      annotations:
+        description: '{{`Cluster {{ $labels.cluster_id }} is almost exceeding its pod limit.`}}'
+      expr: (sum(kube_pod_info) by (cluster_id) / sum(kube_node_status_capacity_pods) by (cluster_id)) > 0.8
```

Comment:
How does it know this is Management Cluster?

Reply:
those metrics come from kube-state-metrics which is installed on all management clusters. The cluster_id is the management cluster id (i.e. installation name)

Reply:
Technically, we will have to remove cluster_id here as this is not useful anymore but we will refactor the alerts at a later time

Reply:
If you look at the top of the file (need to expand the diff as it's not in the modified section), the encompassing rule has a label `cluster_type: "management_cluster"`, and the Prometheuses are created so that workload cluster Prometheuses have a [rule selector that excludes those](https://github.com/giantswarm/prometheus-meta-operator/blob/1eb1635f79e90095debf436c7d752855ba044c8b/service/controller/resource/prometheus/resource.go#L292).